packages/forms/backend/sirius-components-collaborative-forms/src/main/resources/schema/form.graphqls

```diff
@@ -17,9 +17,17 @@ input PropertiesEventInput {
   objectIds: [ID!]!
 }
 
-union FormEventPayload = ErrorPayload | FormRefreshedEventPayload | SubscribersUpdatedEventPayload | WidgetSubscriptionsUpdatedEventPayload
-
-union PropertiesEventPayload = ErrorPayload | FormRefreshedEventPayload | SubscribersUpdatedEventPayload | WidgetSubscriptionsUpdatedEventPayload
```

Comment:
Why the formatting of this lines have been modified?

Reply:
It's formatted by Prettier, I edited with VSCode but I'll revert to one line then.